Isosurfacing_3/include/CGAL/Dual_contouring_3.h

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2022 INRIA Sophia-Antipolis (France).
+// All rights reserved.
+//
+// This file is part of CGAL (www.cgal.org).
+//
+// $URL$
+// $Id$
+// SPDX-License-Identifier: GPL-3.0-or-later OR LicenseRef-Commercial
+//
+// Author(s)     : Julian Stahl
+//                 Daniel Zint
+
+#ifndef CGAL_DUAL_CONTOURING_3_H
```

Comment:
The guard should contain the entire path. Probably the same for all headers.

Reply:
What do you mean by entire path? As far as I've seen the guard is always the path of the file relative to the include directory of the package.

Reply:
Also the filename must be lowercase.